frontend/public/user.js

```diff
@@ -0,0 +1,16 @@
+const users = [
+  {
+    id: 1,
+    username: 'John Doe',
+    avatar: 'J',
+    bio: 'Software developer and designer',
+  },
```

Comment:
For now, this mock JSON is okay, but afterwords `email` & `password` field might be there.